config/networks/mainnet.json

```diff
@@ -24,11 +24,12 @@
   "version": 23,
   "wif": 170,
   "knownWallets": {
-    "AYCTHSZionfGoQsRnv5gECEuFWcZXS38gs": "ARK Bounty",
-    "AZmQJ2P9xg5j6VPZWjcTzWDD4w7Qww2KGX": "ARK GitHub Bounty",
-    "ALgvTAoz5Vi9easHqBK6aEMKatHb4beCXm": "ARK Shield",
-    "AUDud8tvyVZa67p3QY7XPRUTjRGnWQQ9Xv": "ARK Team",
-    "AWkBFnqvCF4jhqPSdE2HBPJiwaf67tgfGR": "ARK Community Fund",
+    "AYCTHSZionfGoQsRnv5gECEuFWcZXS38gs": "Ark Bounty",
+    "AZmQJ2P9xg5j6VPZWjcTzWDD4w7Qww2KGX": "Ark GitHub Bounty",
+    "ALgvTAoz5Vi9easHqBK6aEMKatHb4beCXm": "Ark Shield",
+    "AUDud8tvyVZa67p3QY7XPRUTjRGnWQQ9Xv": "Ark Team",
+    "AWkBFnqvCF4jhqPSdE2HBPJiwaf67tgfGR": "Ark Community Fund",
+    "ANkHGk5uZqNrKFNY5jtd4A88zzFR3LnJbe": "Ark Hot Wallet",
```

Comment:
@kalgoop would you mind to explain what is "Ark Hot Wallet"?

Reply:
Its our wallet for expenses Juan, its marked same on explorer.

Reply:
OK, thanks